components/nextdoor/actions/create-ad-group/create-ad-group.mjs

```diff
@@ -0,0 +1,129 @@
+import app from "../../nextdoor.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "nextdoor-create-ad-group",
+  name: "Create Ad Group",
+  description: "Creates an ad group based on the input payload for an existing campaign. [See the documentation](https://developer.nextdoor.com/reference/adgroup-create).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    advertiserId: {
+      propDefinition: [
+        app,
+        "advertiserId",
+      ],
+    },
+    campaignId: {
+      propDefinition: [
+        app,
+        "campaignId",
+        ({ advertiserId }) => ({
+          advertiserId,
+        }),
+      ],
+    },
+    name: {
+      description: "The name of the ad group.",
+      propDefinition: [
+        app,
+        "name",
+      ],
+    },
+    placements: {
+      type: "string[]",
+      label: "Placements",
+      description: "The placements for the ad group.",
+      options: [
+        "RHR",
+        "FEED",
+        "FSF",
+      ],
+    },
+    bid: {
+      type: "object",
+      label: "Bid",
+      description: "The bid for the ad group.",
+      default: {
+        amount: "USD 10",
+        pricing_type: "CPM",
+      },
+    },
+    budget: {
+      type: "object",
+      label: "Budget",
+      description: "The budget for the ad group. Both `amount` and `budget_type` properties are required.",
+      default: {
+        amount: "USD 1000",
+        budget_type: "DAILY_CAP_MONEY",
+      },
+    },
+    startTime: {
+      propDefinition: [
+        app,
+        "startTime",
+      ],
+    },
+    endTime: {
+      propDefinition: [
+        app,
+        "endTime",
+      ],
+    },
+    frequencyCaps: {
+      type: "string[]",
+      label: "Frecuency Caps",
+      description: "The frecuency caps for the ad group. Where `max_impressions`, `num_timeunits`, and `timeunit` are required. Eligible `timeunit` values are `MINUTE`, `MINUTES`, `HOUR`, `HOURS`, `DAY`, `DAYS`, `WEEK`, `WEEKS`, `MONTH`, `MONTHS`.",
```

Comment:
Shouldn't this be spelled `frequency` in the label and description?

Is it possible to split this into an individual prop for each of 'max impressions', 'num_timeunits' and 'timeunits'? Or can the object have more properties than these three?